packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -17,6 +23,14 @@ import { SortDirection, SortModel } from './sortModel';
  * Set of icons used in the grid component UI.
  */
 export interface IconsOptions {
+  /**
+   * Icon displayed on the side of the column header title to display the filter input component.
+   */
+  columnMenu?: React.ElementType;
```

Comment:
API convention, components are uppercased.

```suggestion
  ColumnMenu?: React.ElementType;
```